python/postprocessing/modules/jme/jetmetHelperRun2.py

```diff
@@ -6,58 +6,70 @@
                   }
 
-jecTagsDATA = { '2016B' : 'Summer16_07Aug2017BCD_V11_DATA', 
-                '2016C' : 'Summer16_07Aug2017BCD_V11_DATA', 
-                '2016D' : 'Summer16_07Aug2017BCD_V11_DATA', 
-                '2016E' : 'Summer16_07Aug2017EF_V11_DATA', 
-                '2016F' : 'Summer16_07Aug2017EF_V11_DATA', 
-                '2016G' : 'Summer16_07Aug2017GH_V11_DATA', 
-                '2016H' : 'Summer16_07Aug2017GH_V11_DATA', 
-                '2017B' : 'Fall17_17Nov2017B_V32_DATA', 
-                '2017C' : 'Fall17_17Nov2017C_V32_DATA', 
-                '2017D' : 'Fall17_17Nov2017DE_V32_DATA', 
-                '2017E' : 'Fall17_17Nov2017DE_V32_DATA', 
-                '2017F' : 'Fall17_17Nov2017F_V32_DATA', 
-                '2018A' : 'Autumn18_RunA_V19_DATA',
-                '2018B' : 'Autumn18_RunB_V19_DATA',
-                '2018C' : 'Autumn18_RunC_V19_DATA',
-                '2018D' : 'Autumn18_RunD_V19_DATA',
-                } 
-
-jerTagsMC = {'2016' : 'Summer16_25nsV1_MC',
-             '2017' : 'Fall17_V3_MC',
-             '2018' : 'Autumn18_V7b_MC'
+jecTagsDATA = { '2016B'   : 'Summer16_07Aug2017BCD_V11_DATA',
+                '2016C'   : 'Summer16_07Aug2017BCD_V11_DATA',
+                '2016D'   : 'Summer16_07Aug2017BCD_V11_DATA',
+                '2016E'   : 'Summer16_07Aug2017EF_V11_DATA',
+                '2016F'   : 'Summer16_07Aug2017EF_V11_DATA',
+                '2016G'   : 'Summer16_07Aug2017GH_V11_DATA',
+                '2016H'   : 'Summer16_07Aug2017GH_V11_DATA',
+                '2017B'   : 'Fall17_17Nov2017B_V32_DATA',
+                '2017C'   : 'Fall17_17Nov2017C_V32_DATA',
+                '2017D'   : 'Fall17_17Nov2017DE_V32_DATA',
+                '2017E'   : 'Fall17_17Nov2017DE_V32_DATA',
+                '2017F'   : 'Fall17_17Nov2017F_V32_DATA',
+                '2018A'   : 'Autumn18_RunA_V19_DATA',
+                '2018B'   : 'Autumn18_RunB_V19_DATA',
+                '2018C'   : 'Autumn18_RunC_V19_DATA',
+                '2018D'   : 'Autumn18_RunD_V19_DATA',
+                'UL2017B' : 'Summer19UL17_RunB_V5_DATA',
+                'UL2017C' : 'Summer19UL17_RunC_V5_DATA',
+                'UL2017D' : 'Summer19UL17_RunD_V5_DATA',
+                'UL2017E' : 'Summer19UL17_RunE_V5_DATA',
+                'UL2017F' : 'Summer19UL17_RunF_V5_DATA',
+              }
+
+jerTagsMC = { '2016'   : 'Summer16_25nsV1_MC',
+              '2017'   : 'Fall17_V3_MC',
+              '2018'   : 'Autumn18_V7b_MC'
+              'UL2017' : 'Summer19UL17_JRV2_MC',
             }
 
 #jet mass resolution: https://twiki.cern.ch/twiki/bin/view/CMS/JetWtagging
 #nominal, up, down
-jmrValues = {'2016' : [1.0, 1.2, 0.8],
-             '2017' : [1.09, 1.14, 1.04],
-             '2018' : [1.24, 1.20, 1.28]
+jmrValues = { '2016'   : [1.0, 1.2, 0.8],
+              '2017'   : [1.09, 1.14, 1.04],
+              '2018'   : [1.09, 1.14, 1.04], # Use 2017 values for 2018 until 2018 are released
```

Comment:
Why were these reverted to the 2017 values?

Reply:
@camclean 
indeed where changes not intended

questions: 
1. where the 2018 come from ? are not in the wiki page
https://twiki.cern.ch/twiki/bin/view/CMS/JetWtagging#2018_scale_factors_and_correctio
2. the original,up,down numbers seems not correct looks more "nominal, down, up"

Reply:
The numbers are working point dependent and are shown in the tables in the section you linked. You can see the JMS and JMR values in the m and sigma column in the data/MC row.

Yes, it should be nominal, down, up.

Reply:
So in the end it was good that this was removed because it was buggy. But the values should now be updated to 2018 values.

Since there are actually multiple working points for W-tagging (and they should be optional), I can make a PR that makes the JMS/JMR values default to 1.0 but have optional other inputs. I think there are a couple of other outstanding updates/fixes from the JME side so maybe I should prepare a presentation for the next XPOG meeting?